control-plane-operator/controllers/hostedcontrolplane/v2/cvo/deployment.go

```diff
@@ -291,3 +298,57 @@ func discoverCVOReleaseImages(cpContext component.WorkloadContext) (string, stri
 
 	return controlPlaneReleaseImage, dataPlaneReleaseImage, nil
 }
+
+func cvoBootstrapScript(clusterID string) string {
+	payloadDir := "/var/payload"
+	enabledCaps := sets.New[configv1.ClusterVersionCapability](
+		configv1.ClusterVersionCapabilitySets[configv1.ClusterVersionCapabilitySetCurrent]...)
```

Comment:
we need to ensure that ClusterVersionCapabilitySetCurrent is actually pointing to the current release - right now it's pointing to 4.15 in the vendor folder - meaning, we also have to backport this